vernac/vernacexpr.ml

```diff
@@ -148,6 +148,7 @@ type record_field_attr = {
   rf_subclass: instance_flag; (* the projection is an implicit coercion or an instance *)
   rf_priority: int option; (* priority of the instance, if relevant *)
   rf_notation: decl_notation list;
+  rf_not_canonical: unit option;
```

Comment:
can't we just have a bool here, and maybe remove the negation (having `true` as default)?

Reply:
Done.